docs/data/data-grid/filtering/customization.md

```diff
@@ -1,119 +1,6 @@
-# Data Grid - Filtering
```

Comment:
> Cons:
> 1. Not sure if two-level nesting is a problem (haven't seen it for any other section)

Do you mean `Data Grid > Filtering > ...`? We already do this for Columns, Rows, Selection pages

Reply:
I mean the path set in [pages.ts](https://github.com/mui/mui-x/blob/5172a58a68e794dc9da87e15d292cf434567fc83/docs/data/pages.ts) file.

It's always something like `/x/package-name/feature-name` but in this PR, I am using something like `/x/package-name/feature-name/sub-feature-name`

Reply:
I don't foresee any issues with the `/feature-name/sub-feature-name` URL, and it looks nicer to me 👍

When I was splitting the Editing page in https://github.com/mui/mui-x/pull/7772, I didn't think much about URLs.
My main motivation was to keep the demos of each subpage in a separate folder (`docs/data/data-grid/*`).
Currently, all filtering-related demos are in the same folder, which makes it a bit harder to navigate (but easier to review this PR 😅).
It's not a big deal though.